.github/workflows/release.yml

```diff
@@ -33,27 +33,13 @@ jobs:
       - name: Build with Maven
         run: ./mvnw --batch-mode clean verify
       - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
+        uses: softprops/action-gh-release@v1
```

Comment:
That is much less code to maintain. Maybe its better that they dropped the maintenance :)

Reply:
Yeah, and finally glob expressions for the file upload! Before that, it was a bit cumbersome to push around step outputs just to get the actual artifact name (in your case the name of the WAR).